hail/src/main/scala/is/hail/types/physical/PCanonicalStruct.scala

```diff
@@ -99,6 +99,10 @@ final case class PCanonicalStruct(fields: IndexedSeq[PField], required: Boolean
   def loadField(offset: Code[Long], fieldName: String): Code[Long] =
     loadField(offset, fieldIdx(fieldName))
 
+  def loadField(offset: Long, fieldName: String): Long = {
```

Comment:
don't add this -- it's pretty expensive to do a string lookup to do pointer arithmetic.